README.md

```diff
@@ -19,14 +19,24 @@ So, the all-new goreleaser was born.
 
 ## Usage
 
-Basically, you need to create a `goreleaser.yml` file in the root of your
+```
+go get github.com/goreleaser/releaser
```

Comment:
There exists a shell script at https://raw.githubusercontent.com/goreleaser/get/master/latest which will bring down the latest, tagged version of the tool which could do with documenting here, too. (Especially given thats the method documented, slightly hidden, in the [CI](#wire-it-with-travis-ci) wiring section further down).